movai_core_shared/core/zmq_server.py

```diff
@@ -150,18 +125,55 @@ def stop(self):
         """Stops the server from running."""
         self._running = False
 
-    async def handle_response(self, response: dict) -> dict:
-        """Handles the response before it is sent back to the client.
-
-        Args:
-            response: The response dict.
+    async def startup(self):
+        """A funtion which is called once at server startup and can be used for initializing
+        other tasks.
         """
         pass
 
+
+class ZMQDispatcher(ZMQServer):
+
+    async def handle(self, buffer) -> None:
```

Comment:
maybe we can pass on this class
and move this function to the message_server class
because it's the only class that uses this method